README.md

```
@@ -299,36 +300,33 @@ func (s *Server) fetchUser(ctx context.Context) (*User, error) {
 
-> 💡 **V2 uses 55% less memory** than V1 by storing only fields instead of the entire logger object.
+> 💡 The extra allocations come from defensively cloning fields on `Set`/`Append`/`Get`
+> and from storing them in the context, so callers can never mutate fields after
+> they are stored. If you need the absolute minimum overhead, pass zap fields directly.
 
 <details>
 <summary>📋 Full Benchmark Results</summary>
 
+Measured on an Apple M2 Pro with Go 1.26 and zap v1.28.0:
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)